includes/component/plugin-tracking/class-tyche-plugin-tracking.php

```diff
@@ -147,7 +147,11 @@ public static function reset_tracker_setting( $plugin_short_name ) {
 		 * Called when the dismiss icon is clicked on the notice.
 		 */
 		public function dismiss_notice() {
-			if ( ! current_user_can( 'manage_options' ) ) {
+			if ( ! is_user_logged_in() ) {
+				return;
+			}
+			$current_user = wp_get_current_user();
+			if ( ! in_array( 'administrator', $current_user->roles, true ) || ! current_user_can( 'manage_options' ) ) {
```

Comment:
@nidhitatosaniya Remove the condition: `! in_array( 'administrator', $current_user->roles, true )`. It isn't necessary.